frontend/src/pages/blog/CreateArticle.js

```diff
@@ -51,15 +52,19 @@ const CreateArticle = () => {
 
   const handleFileChange = (event) => {
     const file = event.target.files[0];
-    console.log(file);
-    console.log(file.text());
-    file.text().then((t) => {
-      const lines = t.split("\n");
-      console.log(lines);
-      setTitle(lines[0]);
-      setSummary(lines[1]);
-      setBody(lines.slice(3).join("\n"));
-    });
+    console.log(file.size)
+    console.log(Config.FILE_SIZE_MAX)
+    if (file.size < Config.FILE_SIZE_MAX) {
```

Comment:
And I think here, we should hard cap it to the max value of 40k bytes to be sure editing the config file don't cause a problem

Reply:
Again I don't get it. You want to hard code the maximum? So what's the point of the config file?

Reply:
You are basically assuming that the config file is correct, but you should not for obvious reason.

The maximum (40Kb) is mostly there so no error happens, but as of right now if you were to put in the config file a maximum size of 300Kb for example, it would pass the test but still generates the error, effectively fixing nothing.

So the test should be something like `if (file.size < Config.FILE_SIZE_MAX && file.size < 40000)`.

And you may also want a different maximum size in the config file to limit space/ressource usage - I don't think a lot of people are going to type 30Kb of text.

Reply:
Since we had the conversation and the max value  before an error is commented somewhere,  the bare minimum is there, so I'm fine with closing this (and will do that).